rabbit-escape-engine/test/rabbitescape/engine/solution/TestSolutionCommand.java

```diff
@@ -13,8 +13,8 @@
     @Test
     public void Equal_solution_commands_are_equal()
     {
-        SolutionCommand solution1 = makeSolutionCommand( 3 );
-        SolutionCommand solution2 = makeSolutionCommand( 3 );
+        SolutionCommand solution1 = makeSolutionCommand();
+        SolutionCommand solution2 = makeSolutionCommand();
 
         assertThat( solution1, equalTo( solution2 ) );
```

Comment:
Should add a similar tests for commands that only contain waits if they don't already exist